sage-ios/SAGE/SAGE/AdminOperations.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  AdminOperations.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 11/11/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+import AFNetworking
+
+class AdminOperations {
+    
+    static func loadMentors(completion: ((NSMutableArray) -> Void)){
```

Comment:
you want to also pass in a failure handler which will take in a string (an error message). just call the failure handler and pass in a custom message in the afnetworking failure block. this should be this way for every request actually